GetStream/stream-chat-android
# [3462] Snapshot tests on CI

### 🎯 Goal

Closes #3462 

Ensure snapshot tests executed locally and on CI produce more or less the same results.

### 🛠 Implementation details

- Added the missing `-no-window` param that prevented snapshot test from execution on CI
- Changed emulator architecture from `x86` to `x86_64` to produce more consistent results with the local script

### 🧪 Testing

1. Fork the repository and merge this feature branch into `main`
2. Add `workflow_dispatch`  trigger to "Verify snapshots" action
3. Execute the action and ensure that it completes successfully

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist
